Assets/Scripts/Weapons/Weapon.cs

```diff
@@ -46,8 +53,26 @@ public bool Attack(Vector3 position, Vector3 direction)
         if (Time.time - lastShot > _fireRate && !_reloading)
         {
             if (_currentMagazine > 0)
-            {
-                _attackBehaviour.initiateAttack(position, direction);
+            {                
+                List<(Vector3, float)> attackDirections = new List<(Vector3, float)>();
+                attackDirections.Add((direction, 0f));
+                
+                if (firingBehavior[PowerUpType.SHOTGUN] || firingBehavior[PowerUpType.REPEATER])
```

Comment:
I'm guessing this is because this is within scope and we aren't looking out more